web/src/app/schedules/ScheduleRuleForm.js

```diff
@@ -103,45 +103,16 @@ const query = gql`
 
 @withStyles(styles)
```

Comment:
Instead of `@withStyles`, you can use `makeStyles` to make a hook for the classes. Here's an example: https://github.com/target/goalert/blob/master/web/src/app/schedules/ScheduleShiftList.js#L54